Fix pam_appl.h include path for Leopard and earlier.
Include this patch only on Leopard or earlier.
Reverts https://github.com/gobby/libinfinity/commit/7921fd451db3cce7fd962080bbedc7b8648be4e9
--- infinoted/infinoted-pam.c.orig	2020-12-11 15:05:36.000000000 -0600
+++ infinoted/infinoted-pam.c	2023-08-01 14:51:00.000000000 -0500
@@ -26,7 +26,11 @@
 #ifdef LIBINFINITY_HAVE_PAM
 
 #include <infinoted/infinoted-pam.h>
+#ifdef __APPLE__
+#include <pam/pam_appl.h>
+#else
 #include <security/pam_appl.h>
+#endif
 #include <sys/types.h>
 #include <grp.h>
 #include <pwd.h>
